cmd/snap/cmd_prepare_image.go

```diff
@@ -88,6 +89,8 @@ For preparing classic images it supports a --classic mode`),
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"revisions": i18n.G("Specify a seeds.manifest file referencing the exact revisions of the provided snaps which should be installed"),
 			// TRANSLATORS: This should not start with a lowercase letter.
+			"write-revisions": i18n.G("Writes a seed.manifest file containing the exact snap revisions used for the image, optionally at the specified path."),
```

Comment:
```suggestion
			"write-revisions": i18n.G("Writes a manifest file containing references to the exact snap revisions used for the image. A path for the manifest is optional."),
```

Difficult to keep this short - and your suggestion was already good. I've just tweaked it to be more inline with `revisions` and also to avoid the potential confusion that the snap revisions can have an optional path.

Reply:
Thank you for the feedback! Updated the text to your suggestion!